suricata.yaml.in

```diff
@@ -731,6 +731,9 @@ af-packet:
     #copy-iface: eth1
     #  For eBPF and XDP setup including bypass, filter and load balancing, please
     #  see doc/userguide/capture-hardware/ebpf-xdp.rst for more info.
+    # Disable HW timestamp support. HW timestamps can be unreliable, see
+    # https://redmine.openinfosecfoundation.org/issues/7585
+    #disable-hwtimestamp: true
```

Comment:
This might make sense for a backport, but would `enable-hwtimestamp` be more appropriate for 9 and going forward?  We can of course decide that later and add some backward compatiblity.